docker/requirements.txt

```diff
@@ -0,0 +1,7 @@
+Flask
+scipy
+numpy
+Cython
+torch
+transliterate
+pathlib
```

Comment:
Pathlib is a standard library starting from python3 so should not be needed. Did you encounter any issue with it?

Reply:
To be honest I don't remember, I can try to remove it and check if it still works